app/src/main/java/net/sourceforge/opencamera/preview/Preview.java

```diff
@@ -4468,6 +4472,12 @@ else if( this.phase == PHASE_TAKING_PHOTO ) {
             if( change_user_pref ) {
                 // now save
                 applicationInterface.setVideoPref(is_video);
+
+                // have to stop settings broadcasting because the mode has changed
+                if( applicationInterface.isSoftwareSyncRunning() &&
+                        applicationInterface.getSoftwareSyncController().isLeader() &&
+                        applicationInterface.getSoftwareSyncController().isSettingsBroadcasting() )
+                    ((MainActivity) getContext()).clickedSyncSettings();
```

Comment:
it is not good idea to call click from other code - as in huge projects you can have some logic inside click which has to be executed only when real person interacts with app